relentless/simulate/hoomd.py

```diff
@@ -304,24 +312,34 @@ def __call__(self, sim):
         ------
         ValueError
             If the simulation box dimensions specified by the file is inconsistent
-            with the ensemble volume (for a constant volume simulation).
+            with the ensemble extent (for a constant extent simulation).
 
         """
         with sim.context:
             sim.system = hoomd.init.read_gsd(self.filename,**self.options)
 
-            # check that the boxes are consistent in constant volume sims.
+            # check that the boxes are consistent in constant extent sims.
             if sim.ensemble.V is not None:
                 system_box = sim.system.box
                 box_from_file = numpy.array([system_box.Lx,
                                           system_box.Ly,
                                           system_box.Lz,
                                           system_box.xy,
                                           system_box.xy,
-                                          system_box.yz])
+                                          system_box.yz,
+                                          system_box.dimensions])
                 box_from_ensemble = self.extract_box_params(sim)
```

Comment:
This would be a possible use of the syntax above:
```suggestion
                box_size, dim = self.extract_box_params(sim)
                if not numpy.all(numpy.isclose(box_from_file,box_size)) or system_box.dimensions != dim:
```
The thing you want to avoid is including `system_box.dimensions` in the array above. Technically, it works, but it is a little confusing because the dimension is an integer but the other values are floats, so they don't really belong in the same floating-point array.